solarforecastarbiter/io/api.py

```diff
@@ -571,6 +572,73 @@ def get_observation_time_range(self, observation_id):
             maxt = maxt.tz_localize('UTC')
         return mint, maxt
 
+    def _get_obs_gaps(self, observation_id, start, end):
+        req = self.get(f'/observations/{observation_id}/values/gaps',
+                       params={'start': start,
+                               'end': end})
+        gaps = req.json()
+        out = []
+        for g in gaps['gaps']:
+            tstamp = pd.Timestamp(g['timestamp'])
+            nextstamp = pd.Timestamp(g['next_timestamp'])
+            # results should never be null, but skip anyway
+            if pd.isna(tstamp) or pd.isna(nextstamp):
+                continue  # pragma: no cover
+            if tstamp.tzinfo is None:
```

Comment:
doesn't the API always return UTC?

Reply:
it should. this is just consistent w/ other methods that parse timestamps